src/main/java/com/qouteall/immersive_portals/portal/Portal.java

```diff
@@ -49,7 +49,9 @@
     public Quaternion rotation;
     
     public double motionAffinity = 0;
-    
+
+    public boolean interactable = true;
```

Comment:
I would recommend making this private and adding a setter method, to discourage developers (especially third party ones using this mod as an API) from accessing it directly.